ovh/resource_cloud_project_instance_test.go

```diff
@@ -106,6 +106,66 @@ func TestAccCloudProjectInstance_basic(t *testing.T) {
 	})
 }
 
+func TestAccCloudProjectInstance3AZ_basic(t *testing.T) {
+	serviceName := os.Getenv("OVH_CLOUD_PROJECT_SERVICE_TEST")
+	region := os.Getenv("OVH_CLOUD_PROJECT_REGION_TEST")
```

Comment:
could you use another env variable, like `OVH_CLOUD_PROJECT_3AZ_REGION_TEST` ? Since this var is already used by many tests, updating it to use a 3AZ version will break our workflows